src/platform/user/authentication/config/constants.js

```diff
@@ -1,4 +1,6 @@
 /* eslint-disable camelcase */
+import { CLIENT_IDS } from '../../../utilities/oauth/constants';
+import { EXTERNAL_APPS } from '../constants';
```

Comment:
Need to ask platform about these imports. Why weren't they there?

Reply:
Yes. At the time of creation there were ESLint warnings about circular references.

Reply:
@asg5704 And to be clear, with circular reference no longer a thing, the import is preferred?

Reply:
I don't believe so